# 坐标系配置
frames:
  # 传感器阵列的父坐标系
  parent_frame: "world"
  # 传感器阵列的基准坐标系
  array_frame: "sensor_array"
  # 传感器阵列相对于父坐标系的位姿
  array_pose:
    position: [0.0, 0.0, 0.0]  # [x, y, z]
    orientation: [0.0, 0.0, 0.0]  # [roll, pitch, yaw] in radians

sensors:
  # - id: 1
  #   position: [0.0, 0.4, 0.0]
  #   orientation: [0.0, 0.0, 0.0]  # RPY格式 (单位: rad)
  - id: 2
    position: [0.1, 0.4, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 3
    position: [0.2, 0.4, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 4
    position: [0.3, 0.4, 0.0]
    orientation: [0.0, 0.0, 0.0]
  # - id: 5
  #   position: [0.4, 0.4, 0.0]
  #   orientation: [0.0, 0.0, 0.0]
  - id: 6
    position: [0.0, 0.3, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 7
    position: [0.1, 0.3, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 8
    position: [0.2, 0.3, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 9
    position: [0.3, 0.3, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 10
    position: [0.4, 0.3, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 11
    position: [0.0, 0.2, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 12
    position: [0.1, 0.2, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 13
    position: [0.2, 0.2, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 14
    position: [0.3, 0.2, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 15
    position: [0.4, 0.2, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 16
    position: [0.0, 0.1, 0.0]
    orientation: [0.0, 0.0, 0.0]
  # - id: 17
  #   position: [0.1, 0.1, 0.0]
  #   orientation: [0.0, 0.0, 0.0]
  - id: 18
    position: [0.2, 0.1, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 19
    position: [0.3, 0.1, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 20
    position: [0.4, 0.1, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 21
    position: [0.0, 0.0, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 22
    position: [0.1, 0.0, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 23
    position: [0.2, 0.0, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 24
    position: [0.3, 0.0, 0.0]
    orientation: [0.0, 0.0, 0.0]
  - id: 25
    position: [0.4, 0.0, 0.0]
    orientation: [0.0, 0.0, 0.0]